aci/resource_aci_fvrsdomatt.go

```diff
@@ -237,6 +238,11 @@ func resourceAciDomain() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+
+			"enhanced_lag_policy": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Shouldn't it have computed ?

Reply:
We already know the value of "enhanced_lag_policy" before applying the configuration, so it should not have Computed.